# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/mine/helptext.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./mine.c~	2005-08-09 12:57:29.000000000 +0200
+++ ./mine.c	2005-09-19 18:03:50.000000000 +0200
@@ -122,7 +122,7 @@
 "E.g. this will create a GEM file from the mine package installed in /:\n"
 "\n"
 "      mine -T /var/adm / mine mine.tar.bz2\n"
-"      mine -C /var/adm mine.tar.bz2 mine.gem\n"
+"      mine -C /var/adm mine.tar.bz2 mine mine.gem\n"
 "\n"
 "Hint: After installing, updating or removing packages you should run the\n"
 "      scripts 'cron.run' and 'postinstall'. This updates some important\n"
